app/data/team.js

```diff
@@ -88,7 +89,14 @@ class Team {
      * @returns {Number} representing the overall quality of the team
      */
     getTeamOverallQuality(){
-        return 0;
+        let calidadEquipo = 0;
```

Comment:
En inglés, y usa const, por que en realidad aunque su valor cambie, siempre será un número, entonces se usa const.
```suggestion
        const teamQuality = 0;
```

Reply:
Este cambio se ha hablado en grupo y se ha resuelto